l10n_it_ricevute_bancarie/models/riba.py

```diff
@@ -34,6 +33,12 @@ def _compute_payment_ids(self):
                 move_lines |= line.payment_ids
             riba.payment_ids = move_lines
 
+    def _compute_total_amount(self):
```

Comment:
C'è qualcosa che non torna nel compute perchè i valori dello scoperto rimangono sempre a 0:

![image](https://github.com/user-attachments/assets/366ffd60-69ce-4d24-9b45-97d11d55783c)

Reply:
controlliamo

Reply:
@odooNextev  il problema risiede in questa linea 
https://github.com/OCA/l10n-italy/blob/38f1d8650d4fc055f9ff888e1d40078b05f2a619/l10n_it_ricevute_bancarie/models/account.py#L67
dato che l'uso di `display_type` non è lo stesso che se ne fa nella 16.0 non fa partire il calcolo.

Hai qualche suggerimento con cosa si possa sostituire?

Reply:
display_type = "payment_term" deriva da questa condizione: https://github.com/odoo/odoo/blob/16.0/addons/account/models/account_move_line.py#L440
potresti replicarla

Reply:
@odooNextev credo che la versione riadattata sia
```
and line.account_id.internal_type in ['receivable', 'payable']
```
solo che cosi facendo se una riba è insoluta il campo `open_amount` non viene calcolato a 0

Reply:
> solo che cosi facendo se una riba è insoluta il campo open_amount viene calcolato a 0

@odooNextev ping, se hai qualche suggerimento è ben accolto!

Reply:
Problema risolto, tutto stava nei termini di pagamento che nei dati di demo della 14.0, non toglieva la linea aggiunta di default quando si creava il termine di pagamento. Di conseguenza non era configurato nel modo giusto il termine che non impostava la corretta data di scadenza sulla RiBa. Ho aggiornato i dati e ora il calcolo di `open_amount` è corretto anche se segnata come insoluta.

Reply:
Adesso funzionano entrambe allo stesso modo: https://www.loom.com/share/af6b1e3d253a4e8896fb6286a55cd64c

@odooNextev puoi verificare? Grazie!

Reply:
@stenext ping :)